src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is working.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working.");
```

Comment:
The message in 'stopWork' method should also be more informative. Consider specifying the machine type like 'Excavator stopped working' to make the logs more understandable.